jobqueue/src/main/java/com/path/android/jobqueue/JobManager.java

```diff
@@ -163,7 +163,8 @@ private int countReadyJobs(boolean hasNetwork) {
     /**
      * Adds a new Job to the list and returns an ID for it.
      * @param job to add
-     * @return id for the job.
+     * @return id for the job. If another job with the same singleId was already queued and not running,
```

Comment:
this has to change because there are no `long` ids anymore. We can discuss later once moved into message-queue branch